seed_embeddings/OpenKE/generate_embedding_ray.py

```diff
@@ -172,21 +206,43 @@ def findRep(src, dest, index_dir):
         print("Error in files")
         raise Exception("Error in files")
 
+    scheduler = ASHAScheduler(
+        time_attr="training_iteration",
+        max_t=arg_conf.epoch,
+        grace_period=min(arg_conf.epoch, 4000),
+        reduction_factor=2,
+        metric="hit1",
+        mode="max",
+    )
+
     tuner = tune.Tuner(
         train,
         param_space=search_space,
-        tune_config=TuneConfig(max_concurrent_trials=4),
+        tune_config=TuneConfig(
+            max_concurrent_trials=4,
+            scheduler=scheduler,
+            num_samples=8,
+        ),
+        run_config=RunConfig(
+            checkpoint_config=CheckpointConfig(
+                num_to_keep=2,
+                # *Best* checkpoints are determined by these params:
```

Comment:
These values are to be set right? We can check if `is_analogy` is set and select based on that.